packages/frontend/src/components/chat.tsx

```diff
@@ -57,9 +57,9 @@ export default function ChatContainer() {
       console.log(`${topic}: ${msg}`)
 
       if (evt.detail.type === 'signed') {
-        const peerId = evt.detail.from.toString()
+        setMessages([...messages, { msg, from: 'other', peerId: evt.detail.from.toString() }])
       } else {
-        setMessages([...messages, { msg, from: 'other', peerId: 'unkonwn' }])
+        const peerId = evt.detail.toString()
```

Comment:
Do we even need to store this `peerId` if the message is unsigned?

Reply:
No I don't see why we'd need to. I think we can just discard unsigned messages. I'll delete it